ai-assistants/loaders/messages.ts

```diff
@@ -0,0 +1,48 @@
+import { ReplyMessage } from "../actions/chat.ts";
+import {
+  MessageContentImageFile,
+  MessageContentText,
+  ThreadMessage,
+} from "../deps.ts";
+import { AppContext } from "../mod.ts";
+export interface Props {
+  thread: string;
+  after?: string;
+  before?: string;
+}
+
+const normalize = (strContent: string) => {
+  return strContent.endsWith("@") || strContent.endsWith("#")
+    ? strContent.slice(0, strContent.length - 2)
+    : strContent;
+};
```

Comment:
extract POSITIVE_TOKEN (@) and NEGATIVE_TOKEN (#) (use also on `ai-assistants/chat/messages.ts`)